modules/metering-troubleshooting.adoc

```diff
@@ -10,10 +10,173 @@ A common issue with metering is pods failing to start. Pods might fail to start
+hive-metastore-0                      2/2     Running   0          45h
+hive-server-0                         3/3     Running   0          45h
+metering-operator-7b9d7958d7-z6n46    2/2     Running   0          46h
+presto-coordinator-0                  2/2     Running   1          45h
+reporting-operator-84f7c9b7b6-fr697   2/2     Running   542        8d <1>
+----
+<1> The `reporting-operator` pod is restarting at a high rate.
+
+. Check the `reporting-operator` pod for an `OOMKilled` termination:
++
+[source,terminal]
+----
+$ oc describe pod/reporting-operator-84f7c9b7b6-fr697
+----
++
+.Example output
+[source,terminal]
+----
+Name:         reporting-operator-84f7c9b7b6-fr697
+Namespace:    operator-metering
+Priority:     0
+Node:         ip-10-xx-xx-xx.ap-southeast-1.compute.internal/10.xx.xx.xx
+...
+   Ports:          8080/TCP, 6060/TCP, 8082/TCP
+   Host Ports:     0/TCP, 0/TCP, 0/TCP
+   State:          Running
+     Started:      Thu, 03 Dec 2020 20:59:45 +1000
+   Last State:     Terminated
+     Reason:       OOMKilled <1>
+     Exit Code:    137
+     Started:      Thu, 03 Dec 2020 20:38:05 +1000
+     Finished:     Thu, 03 Dec 2020 20:59:43 +1000
+----
+<1> The `reporting-operator` pod was terminated due to OOM kill.
+
+[discrete]
+[id="metering-check-and-increase-memory-limits_{context}"]
+=== Increasing the reporting-operator pod memory limit
+
+If you are experiencing an increase in pod restarts and OOM kill events, you can check the current memory limit set for the `reporting-operator` pod. Increasing the memory limit allows the `reporting-operator` pod to update the report data sources. If necessary, increase the memory limit in your `MeteringConfig` resource by 25-50%.
+
+.Procedure
+
+. Check the current memory limits of the `reporting-operator` pod:
++
+[source,terminal]
+----
+$ oc describe pod reporting-operator-67d6f57c56-79mrt
+----
++
+.Example output
+[source,terminal]
+----
+Name:         reporting-operator-67d6f57c56-79mrt
+Namespace:    operator-metering
+Priority:     0
+...
+   Ports:          8080/TCP, 6060/TCP, 8082/TCP
+   Host Ports:     0/TCP, 0/TCP, 0/TCP
+   State:          Running
+     Started:      Tue, 08 Dec 2020 14:26:21 +1000
+   Ready:          True
+   Restart Count:  0
+   Limits:
+     cpu:     1
+     memory:  500Mi <1>
+   Requests:
+     cpu:     500m
+     memory:  250Mi
+   Environment:
+...
+----
+<1> The current memory limit for the `reporting-operator` pod.
+
+. Edit the `MeteringConfig` resource to update the memory limit:
++
+[source,terminal]
+----
+$ oc edit meteringconfig/operator-metering
+----
++
+.Example `MeteringConfig` resource
+[source,yaml]
+----
+  kind: MeteringConfig
+  metadata:
+  name: operator-metering
+   namespace: operator-metering
+  spec:
+   openshift-reporting:
+     spec:
+       awsBillingReportDataSource:
+         bucket: gmeteringbucket
+         enabled: true
+         region: ap-southeast-1
```

Comment:
Should we remove this configuration? Would that be confusing if we explicitly document you need to configure storage, but omit those requisite properties from a MeteringConfig example?

Reply:
@timflannagan 
Can that section be more generic for the example? 
```
       BillingReportDataSource:
         bucket: meteringbucket
         enabled: true
         region: southeast-1
```

Reply:
I realize we're configuring AWS billing here, and not the storage configuration, so we can omit this section entirely.